### PR TITLE
Print info related to starting and finishing modules

### PR DESCRIPTION
**Pull request by dnkolegov**
This PR adds printing log messages related to starting and finishing modules. The log level is INFO. That can be changed in the future to DEBUG.

### CODE REVIEW

File: node.go [hunk 1]
```diff
@@ -127,6 +129,8 @@ func NewNode(
 		inputPausedCond: sync.NewCond(&sync.Mutex{}),
 
 		stopped: make(chan struct{}),
+
+		logger: config.Logger,
```
Comment (matejpavlovic) [inline node.go:133]:
```suggestion
```

File: node.go [hunk 2]
```diff
@@ -244,6 +248,8 @@ func (n *Node) processWAL(ctx context.Context) error {
 // which mostly consists of routing events between the node's modules.
 // Stops and returns when ctx is canceled.
 func (n *Node) process(ctx context.Context) error { //nolint:gocyclo
+	n.logger.Log(logging.LevelInfo, "node process started")
+	defer n.logger.Log(logging.LevelInfo, "node process finished")
```
Comment (matejpavlovic) [inline node.go:252]:
```suggestion
	n.Config.logger.Log(logging.LevelInfo, "node process started")
	defer n.Config.logger.Log(logging.LevelInfo, "node process finished")
```

**Review by matejpavlovic — CHANGES_REQUESTED** [review]
The logger is already passed as part of `NodeConfig`.
Otherwise, perfect. :-)

File: node.go [hunk 3]
```diff
@@ -82,6 +82,8 @@ type Node struct {
 	// This channel is closed by the Run and Debug methods on returning.
 	// Closing of the channel indicates that the node has stopped and the Stop method can also return.
 	stopped chan struct{}
+
+	logger logging.Logger
```
[the remark this answers is not in the document]
Comment (dnkolegov) [reply node.go]: Fixed

**Review by matejpavlovic — APPROVED** [review]
Oh yes, I forgot the capital L...
Feel free to merge.
Thanks!

**Pull request merged**